src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -235,19 +232,38 @@ const DropDownMenu = React.createClass({
     }
   },
 
+  handleKeyDown(event) {
+    switch (event.keyCode) {
+      case 13:
+      case 32:
+      case 40:
+        event.preventDefault();
```

Comment:
Could we use `keycode` here? (https://github.com/timoxley/keycode)

It was added to other components recently.